### ISSUE TITLE
Cursor gets stuck while scrolling down in Hexdump-View 

### ISSUE THREAD

**Issue by Semnodime** [rizinorg/cutter#2553]
**Environment information**
* Operating System: Linux Mint 20
* Cutter version:
Version 1.12.0
Using rizin-0.1.0-git
Based on Qt 5.12.8 (GCC 9.3.0, 64 bit)
Optional Features:
Python: OFF
Python Bindings: OFF
**Keyboard: NEO2** (used for `[arrow down]`)

* Obtained from:
  - [x] Built from source
  - [ ] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format: ELF

**Describe the bug**
*Sometimes* (sadly randomly as far as I can tell) the curser gets stuck while scrolling down in the hexdump view pressing the `[arrow down]` key.
This only ever happens when scrolling down and when using the arrow key `[down]`.
Once this bug occurs, I can navigate the cursor up and back down and it will get stuck on the same line:
The cursor snaps to the end of the line, as if it was the end of the file (though it clearly is not).

Interestingly, pressing and holding `[mouse left button]` into the hexdump view, then additionally moving the cursor down outside the hexdump view frame (which also leads to scroll event) works **and fixes the stuck position**. The same appears to be iithe case when using `[page down]` instead of `[arrow down]`. After scrolling past the problematic spot, the arrow keys will work again as expected, even when scrolling back up and down over the same position.

**To Reproduce**

(random occurence, still notice the following circumstance)
1. Open an ELF binary
2. View its hexdump
3. Scroll with the `[arrow down]` key (maybe switch into the disasm view now and then (or simulate normal usage of cutter)
4. Get your cursor stuck on a line during downwards scroll

**Expected behavior**
Scrolling without getting stuck.

**Screenshots**
Before pressing and holding `[arrow down]` for scrolling
![image](https://user-images.githubusercontent.com/19511852/103721917-230fe180-4fcf-11eb-9dad-23e58825e16b.png)
After pressing and holding `[arrow down]`. Notice how the cursor also moves to the end of the line, as if the file ended or such.
![image](https://user-images.githubusercontent.com/19511852/103722188-b517ea00-4fcf-11eb-98bb-33e1ce9ae1cd.png)

**Comment by Semnodime** [comment]
**Additional Information**
Maybe having the graph view opened side-by-side is causing the problem, as trying to load the graph view might block or fault.

**Comment by Semnodime** [comment]
Yep! Can confirm, that closing the graph overview pane will fix a position that has just been stuck before closing the pane.
I even moved the graph overview above the hexdump instead of side-by-side, to make sure it does not reload the hexdump by resizing and similar.

**Comment by karliss** [comment]
Doesn't seem to be graph related. Was able to repeat it without graph view and with seek synchronization disabled. Looks like it happen on round values every 0x1000 bytes.
